source/slang/slang-emit-cpp.cpp

```diff
@@ -2476,6 +2508,33 @@ bool CPPSourceEmitter::tryEmitInstExprImpl(IRInst* inst, const EmitOpInfo& inOut
             }
             return true;
         }
+        case kIROp_MakeExistential:
+        case kIROp_MakeExistentialWithRTTI:
+        {
+            auto rsType = cast<IRComPtrType>(inst->getDataType());
```

Comment:
There is the irriting issue around COM interfaces where if I have 

```
interface IA : IUnknown {}
interface IB : IUnknown {}
class C : IA, IB {}

C c;

// Fails, because it's not clear which IUnknown is wanted, IAs, or IBs. 
IUnknown* ptr = &c;

// So in practice you have to static cast to a specific derived interface that implements your interface. For example
IUnknown* ptr = static_cast<IA*>(&c);

```

Reply:
This will be one of the issues when we get to deal with transitvely inherited interfaces.